overlord/hookstate/ctlcmd/refresh_test.go

```diff
@@ -46,10 +46,9 @@ type refreshSuite struct {
 
 var _ = Suite(&refreshSuite{})
 
-func mockRefreshCandidate(snapName, instanceKey, channel, version string, revision snap.Revision) interface{} {
+func mockRefreshCandidate(snapName, channel, version string, revision snap.Revision) interface{} {
 	sup := &snapstate.SnapSetup{
-		Channel:     channel,
-		InstanceKey: instanceKey,
```

Comment:
That got me confused, I reckon it's because of empty "" in the tests.

Reply:
Exactly, the parameter was only used to pass "" so InstanceKey was never really changed